tests/Item/MenuItem/MenuItemSortTest.php

```diff
@@ -0,0 +1,94 @@
+<?php declare(strict_types=1);
+
+namespace Tests\Becklyn\Menu\Item\MenuItem;
+
+use Becklyn\Menu\Exception\InvalidSortMethodException;
+use Becklyn\Menu\Item\MenuItem;
+use Becklyn\Menu\Sorter\MenuItemSorter;
+use PHPUnit\Framework\TestCase;
+
+class MenuItemSortTest extends TestCase
+{
+    private function buildMenu (string $sort) : MenuItem
+    {
+        $root = new MenuItem(null, ["sort" => $sort]);
```

Comment:
Can we also get a test-case with some special chars numbers and one with the same letter in capital and lower case?

Reply:
Not sure about what you mean by „special chars numbers“.

Capital letters are already included, the important thing to test (= test category) is the case insensitivity, not the actual letters (= would be no new test category).